sympy/printing/pretty/stringpict.py

```diff
@@ -7,9 +7,13 @@
     - terminal_string() method
     - minor fixes and changes (mostly to prettyForm)
 
-TODO:
-    - Allow left/center/right alignment options for above/below and
-      top/center/bottom alignment options for left/right
+Updates by Juan Mauricio Matera (email <my last name> at ar_ude_plnu_acisif):
+    - support for align attribute in `left`, `right`, `above` and `below`
+      methods
+    - adding the v_align() method
+    - Methods `parens`, `left`, `right`, `above` and `below` now returns
+      objects of the caller object class.
+    - The method `root` now works.
```

Comment:
We should probably just get rid of this docstring. We don't really do these sorts of inline changelogs in SymPy.

Reply:
OK. Gone.